drivers: usdhc: set correct driver strength

lgl88911 opened the pull request:
spec 3.0 support driver strehgth and current limit.
spec 1.10 and after support CMD6.

Signed-off-by: Frank Li <lgl88911@163.com>

lgl88911 committed:
driver: usdhc: fix spec 1.0 sdcard can't work

The SD Card below Physical Layer 1.10 does not support CMD6.
When usdhc_sd_init is executed, it will fail to exit due to
execution of CMD6, causing the SD Card to not work normally.

For different SD Card Physical Layer specifications,
the following modifications have been made:
Version 1.10 and above support CMD6.
Version 3.0 and above support CMD6 Group 3 (driver strength
and current limit) .

Signed-off-by: Frank Li <lgl88911@163.com>

jfischer-no commented on drivers/disk/usdhc.c:
```diff
@@ -2534,20 +2534,22 @@ static int usdhc_sd_init(struct usdhc_priv *priv)
 		usdhc_set_bus_width(base, USDHC_DATA_BUS_WIDTH_4BIT);
```
Please change commit title to "drivers: usdhc: fix sdcard spec 1.0 support"
Please fix commit body and describe the issue or the fix. spec 3.0/spec 1.10 is probably SD card specification?

lgl88911 replied:
Yes, It is Physical Layer Simplified Specification Version.
I have some SD cards with spec 1.0 that can’t work normally under Zephyr. The reason is that the CMD6 operation is performed on it in usdhc, butstart to  spec 1.10 and above support CMD6, which will cause usdhc_sd_initfailure. This patch can let The SD Card of spec 1.0 can work normally.
Since SD Card has multiple spec versions, CMD 6 Group 3 (driver strength and current limit) is only supported after spec 3.0, so it also needs to be restricted.

jfischer-no commented on drivers/disk/usdhc.c:
```diff
@@ -2534,20 +2534,22 @@ static int usdhc_sd_init(struct usdhc_priv *priv)
+		ret = usdhc_select_fun(priv, SD_GRP_DRIVER_STRENGTH_MODE,
+			priv->card_info.driver_strength);
+		if (ret) {
+			LOG_ERR("Set SD driver strehgth failed: %d\r\n", ret);
```
should be `LOG_ERR("Set SD driver strength failed: %d", ret);`, same below

lgl88911 replied:
It seems that all LOG messages have \r\n, I have removed them all

MaureenHelm replied:
There's also a typo in the message, "strength" is misspelled.

lgl88911 committed:
driver: usdhc: modify LOG message

LOG API will auto add \r\n at output end,
remove the \r\n at the end of debug messge.

Fix misspelled.

Signed-off-by: Frank Li <lgl88911@163.com>

jfischer-no reviewed (approved):
Thanks
